src/components/OrganizationListTable.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import { Organization, OrganizationBundle } from '@pdc/sdk';
+import { TableRow, RowCell } from './Table';
+import { ListTable } from './ListTable';
+
+interface OrganizationListTableRowProps {
+	organization: Organization;
+}
+
+const OrganizationListTableRow = ({
+	organization,
+}: OrganizationListTableRowProps) => {
+	const navigate = useNavigate();
+
+	const handleRowClick = () => {
+		// Only navigate if we didn't just finish selecting text.
+		if (window.getSelection()?.isCollapsed) {
```

Comment:
A COMMENT?!?!!  

Just kidding -- but also, if we put this check in a named function could we eliminate the comment?  For instance something like `userHasNotSelectedText()`

That way the code itself reads "if the user has not selected text, then navigate" and you don't need the comment 🎉